index.js

```diff
@@ -62,10 +63,10 @@ Merry.prototype.start = function () {
 }
 
 Merry.prototype.listen = function (port) {
-  assert.equal(typeof port, 'number', 'Merry.listen: port should be type number')
+  this._port = port || process.env.PORT || this.env.PORT
+  assert.ok(this._port, 'Merry.listen: port should exist')
```

Comment:
yea `assert.ok()` just checks for a truthy so this should be good if it's not a Number

(this assert was checking for a number before tho which, yea, doesn't work with process.env stuff cause those come as strings)